api/route.go

```diff
@@ -64,9 +71,16 @@ func handleSubRoute(baseRouter *mux.Router, path string, fn func(router *mux.Rou
 
 func (s *Server) genRouter() http.Handler {
 	router := mux.NewRouter()
-	router.HandleFunc(routePing, s.handlePing)
-	handleSubRoute(router, routeDependencies, s.genDependenciesRouter)
-	handleSubRoute(router, routeInstances, s.genInstancesRouter)
-	handleSubRoute(router, routeProxyConfigs, s.genProxyConfigsRouter)
+	apiRoute := router.PathPrefix(baseRoute).Subrouter()
+	apiRoute.HandleFunc(routePing, s.handlePing)
+	handleSubRoute(apiRoute, routeDependencies, s.genDependenciesRouter)
+	handleSubRoute(apiRoute, routeInstances, s.genInstancesRouter)
+	handleSubRoute(apiRoute, routeProxyConfigs, s.genProxyConfigsRouter)
+
+	statikFS, err := fs.New()
+	if err != nil {
+		logger.Fatal(err)
+	}
+	router.PathPrefix("/").Methods(http.MethodGet).Handler(http.FileServer(statikFS))
```

Comment:
I have some confusions,  http server usually provides a handler which handles the request of static resources, but why we need package these static resources into binary file here?